include/nbl/builtin/hlsl/device_capabilities_traits.hlsl

```diff
@@ -11,6 +11,13 @@
 #include "nbl/builtin/hlsl/enums.hlsl"
 #include "nbl/video/device_capabilities_traits_testers.hlsl"
 
+using ShaderStage = nbl::hlsl::ShaderStage;
+using SampleCountFlags = nbl::hlsl::SampleCountFlags;
+using ResolveModeFlags = nbl::hlsl::ResolveModeFlags;
+using SpirvVersion = nbl::hlsl::SpirvVersion;
+using PointClippingBehavior = nbl::hlsl::PointClippingBehavior;
+using SwapchainMode = nbl::hlsl::SwapchainMode;
```

Comment:
you don't need these typedefs, whatever is using them should also be under the `nbl::hlsl::` namespace